From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 31 May 2010 18:42:26 +0100
Subject: [PATCH] [powerpc] Restore header inclusion to maintain ABI

commit 73ef533de7533fcf514403125a0d4747d3390317 "powerpc/perf_event:
Fix oops due to perf_event_do_pending call" removed use of the
perf_event API from arch/powerpc/kernel/irq.c and also removed the
now-redundant inclusion of <linux/perf_event.h>.  Unfortunately this
changed the symbol magic for several functions exported from this
file.  Restore the header inclusion for now.
---
 arch/powerpc/kernel/irq.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/kernel/irq.c b/arch/powerpc/kernel/irq.c
index 8564a41..a7018de 100644
--- a/arch/powerpc/kernel/irq.c
+++ b/arch/powerpc/kernel/irq.c
@@ -53,6 +53,7 @@
 #include <linux/bootmem.h>
 #include <linux/pci.h>
 #include <linux/debugfs.h>
+#include <linux/perf_event.h>
 
 #include <asm/uaccess.h>
 #include <asm/system.h>
-- 
1.7.1

